public/home.html

```diff
@@ -13,6 +13,10 @@ <h1 class="mb-1 fw-semibold">
   <div id="blockchain-metrics"></div>
 </div>
 
+<div class="container w-95 text-center text-muted">
+  <div class="cursor-grab" id="rpc-url"></div>
```

Comment:
I think there was miscommunication here. 

1. I thought you had tailwind so suggested using their class to get a "grab" cursor: `cursor-grab`
2. Using a class that's referenced makes sense, but in the current implementation the class is called "cursor-grab" but gives a `cursor: pointer`. 
3. I suggested a cursor type of "copy" which might be fun to use as it's closer to the action. https://esqsoft.com/css-help/css-cursor-examples.htm

So, my suggestion would be: 
Call the class "copyable"
make the cursor `pointer` or `copy`

```
.copyable {
  cursor: 'copy';
}
```

Reply:
The bootstrap way to get the same effect that you have currently is simply to put a role="button" on the class.

see https://getbootstrap.com/docs/4.0/components/buttons/#button-tags

> When using button classes on <a> elements that are used to trigger in-page functionality (like collapsing content), rather than linking to new pages or sections within the current page, these links should be given a role="button" to appropriately convey their purpose to assistive technologies such as screen readers.